ci/do_ci.sh

```diff
@@ -467,7 +467,7 @@ case $CI_TARGET in
             --define wasm=wasmtime \
             --define enable_logging=disabled \
             -c fastbuild \
-            @envoy//source/exe:envoy-static \
+            @envoy//contrib/exe:envoy-static \
```

Comment:
Add this change to force the CI test FIPS build with contrib extensions. This should be reverted after the review.

Reply:
@phlax should we add some task to test the FIPS build with contrib extensions? It seems our CI didn't build this case. I also did a test in another PR https://github.com/envoyproxy/envoy/pull/33757, if we build FIPS with contrib extension can expose the issue.

Reply:
@soulxu as mentioned offline - not sure about adding contrib, im concerned that it could add a very big overhead for ~uncached runs - maybe it would be ok - and for sure, it will catch a load of issues that we dont catch in ci and create issues/prs/etc after

Reply:
there is a similar issue with the gcc/contrib build

Reply:
yea, I can understand, if we want to resolve this issue, then we going to add new job in the CI. If the burden can't afford, it is fine for me also.

Reply:
lets do that in a separate PR, and ideally get some timings so we know what the impact is

Reply:
got it